test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -1183,6 +1200,26 @@ Describe "Invoke-WebRequest tests" -Tags "Feature" {
 
     #endregion charset encoding tests
 
+    #region Client Certificate Authentication
+
+    It "Verifies Invoke-WebRequest Certificate Authentication Fails without -Certificate"  {
+        $uri = 'https://localhost:8443/'
+        $result = Invoke-WebRequest -Uri $uri -SkipCertificateCheck
+        $failedMessage = 'Status: Failed'
+        $result.RawContent  | Should Match ([regex]::Escape($failedMessage ))
+    }
+
+    It "Verifies Invoke-WebRequest Certificate Authentication Successful with -Certificate" {
```

Comment:
@iSazonov I'm not opposed to adding the comments, but, shouldn't this be consistent with the other tests which are not documenting external behavior? If I add the comments here, the other tests should require the same. If there are no plans to document every test accessing either HttpListener or httpbin, why should my test be different? When I'm troubleshooting one of the current test failures I have to go to either httpbin or the source for HttpListener to figure out what should be returned. All of these tests are checking the response from a server, to me it makes sense for the documentation of that server to rest with the server and not in the test. We shouldn't document the behavior in 2 places, IMO

As a compromise, what about a comment suggesting to consult the README.md for behavior of the listener?